src/fixtures/logstash_fields.js

```diff
@@ -51,6 +53,10 @@ function stubbedLogstashFields() {
       scripted = !!script,
     } = metadata;
 
+    const type = (esType === 'conflict' || esType === 'unknown')
```

Comment:
`esType` can never be `unknown` here.

Also, what is `conflict` and why is it special-cased here? (instead of being handled within `castEsToKbnFieldTypeName`)

(maybe just add a comment above this line that describes why we're special-casing these?)